app/views/layouts/application.html.haml

```diff
@@ -28,6 +28,52 @@
       = yield
     %footer{role: "contentinfo"}
       .wrapper
-        %span
-          - unless user_signed_in?
-            = link_to('Sign in', new_user_session_path)
```

Comment:
You've removed the sign in link from the footer but not included it anywhere else. We really do need to have it somewhere.